libmisc/btrfs.c

```diff
@@ -0,0 +1,94 @@
+#include <linux/btrfs_tree.h>
+#include <linux/magic.h>
+#include <sys/statfs.h>
+
+#include "prototypes.h"
+
+
+static int run_btrfs_subvolume_cmd(const char *subcmd, const char *arg1, const char *arg2)
+{
+	int status = 0;
+	const char *cmd = "/sbin/btrfs";
```

Comment:
Does that make sense unconditionally? Can't we just pass `btrfs` in case it's not located in `/sbin`.

Reply:
It should be either in /sbin or /usr/sbin. And if in latter, there should be a symlink from /sbin. It should not be in user's PATH. We also need to path for line 26.

Alternatively, the path should be moved to the autoconf scripts.